### PR TITLE
Remove "Learn more" label from localized cloud services PN

### PR DESCRIPTION
**Pull request by kyoshino**
This is a follow-up fix of https://github.com/mozilla/legal-docs/pull/373 by @johngruen. @peiying2 can you take a look and merge? Thanks!

### CODE REVIEW

File: firefox_cloud_services_PrivacyNotice/ta.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ## உங்களுக்கு தெரிய வேண்டியவைகள்:
 
-நீங்கள் பயன்படுத்தும் சேவைகளின் அம்சங்களைப் பொறுத்து எங்களுக்கு பல்வேறு வகையான தரவை அனுப்புகிறீர்கள். மேலும் அறிக.
+நீங்கள் பயன்படுத்தும் சேவைகளின் அம்சங்களைப் பொறுத்து எங்களுக்கு பல்வேறு வகையான தரவை அனுப்புகிறீர்கள்.
```
Comment (TDA) [inline firefox_cloud_services_PrivacyNotice/ta.md]: I can vouch that this is perfect. How on earth do you know so many languages? :O @kyoshino

Comment (TDA) [inline firefox_cloud_services_PrivacyNotice/ta.md]: or did you just remove them based on position? :D you sir are awesome!

Comment (kyoshino) [inline firefox_cloud_services_PrivacyNotice/ta.md]: Just guess :smile_cat:

Comment (kyoshino) [inline firefox_cloud_services_PrivacyNotice/ta.md]: Yup, on line 10 and 18, the same short sentences.

**Comment by johngruen** [comment]
@kyoshino thanks for doing this!

**Comment by kyoshino** [comment]
:+1:

**Comment by peiying2** [comment]
@kyoshino, thank you for the quick fix!!

**Pull request merged**